WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -132,11 +141,15 @@ final class ProductSelectorViewModel: ObservableObject {
          purchasableItemsOnly: Bool = false,
          storageManager: StorageManagerType = ServiceLocator.storageManager,
          stores: StoresManager = ServiceLocator.stores,
+         supportsMultipleSelections: Bool = false,
```

Comment:
Why do we set these default values?

Reply:
Currently we set these to `false` on initialization because is the default way to select products (single-selection), then we either pass `true` on initialization when creating the viewmodels for Coupons, or via if needed via feature flag for Product multi-selection.